<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalData</class>
 <widget class="QWidget" name="PersonalData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>487</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>487</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>487</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Управление учетными записями, правами доступа.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="groupsTab">
      <attribute name="title">
       <string>Группы</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0" rowspan="4">
        <widget class="QTreeView" name="groupsView"/>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="addGroupButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="deleteGroupButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="editGroupButton">
         <property name="text">
          <string>Редактировать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="usersTab">
      <attribute name="title">
       <string>Пользователи</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QPushButton" name="addUserButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="deleteUserButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="editUserButton">
         <property name="text">
          <string>Редактировать</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="5">
        <widget class="QListView" name="usersView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rolesTab">
      <attribute name="title">
       <string>Роли</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" rowspan="6">
        <widget class="QListView" name="rolesView"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="editRoleButton">
         <property name="text">
          <string>Редактировать</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="addRoleButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="deleteRoleButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="promissionsTab">
      <attribute name="title">
       <string>Разрешения</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QPushButton" name="deletePromissionButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="addPromissionButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="4">
        <widget class="QListView" name="promissionsView"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="infoLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
